components/LinkButton.vue

```diff
@@ -13,14 +18,28 @@ export default {
     link: {
       type: String,
       required: true
+    },
+
+    roundedNone: {
+      type: Boolean
+    },
+
+    textColor: {
+      type: String,
+      default: 'text-white'
+    },
+
+    bgColor: {
+      type: String,
+      default: 'bg-gradient-to-r from-primary hover:from-primary-hover hover:to-secondary-hover to-secondary'
+    }
+  },
```

Comment:
Isso me parecem classes e não cores. Recomendo um vetor indexado pelo nome da cor e o resultado as classes que devem ser aplicadas quando aquela cor for escolhida, por exemplo:

```js
const backgroundColors = {
  primary: 'bg-gradient-to-r from-primary hover:from-primary-hover hover:to-secondary-hover to-secondary'
}
```

e no Vue o usuário poderia passar apenas `primary` ou `secondary` para a prop.

``` js
export default {
  props: {
    backgroundColor: {
      default: 'primary',
      type: String
    }
  },

  computed: {
    backgroundColorClasses () {
      return backgroundColors[this.backgroundColor]
    }
  }
}
```

Obs: o mesmo vale para a cor do texto.